enum.c

```diff
@@ -132,21 +132,23 @@ enum_grep0(VALUE obj, VALUE pat, VALUE test)
 }
 
 /*
- *  call-seq:
- *     enum.grep(pattern)                  -> array
- *     enum.grep(pattern) { |obj| block }  -> array
- *
- *  Returns an array of every element in <i>enum</i> for which
- *  <code>Pattern === element</code>. If the optional <em>block</em> is
- *  supplied, each matching element is passed to it, and the block's
- *  result is stored in the output array.
- *
- *     (1..100).grep 38..44   #=> [38, 39, 40, 41, 42, 43, 44]
- *     c = IO.constants
- *     c.grep(/SEEK/)         #=> [:SEEK_SET, :SEEK_CUR, :SEEK_END]
- *     res = c.grep(/SEEK/) { |v| IO.const_get(v) }
- *     res                    #=> [0, 1, 2]
- *
+ * call-seq:
+ *   grep(object) -> new_array
```

Comment:
`pattern` is more descriptive than `object`, on the other hand it might be too narrow (e.g. for Integer); not sure which would be better; personally I would tend to more descriptive, but that is a matter of taste.

Reply:
Alternative might be pattern_or_object, which I think too clunky.  Prefer stet.

Reply:
'pattern' it is.